DESCRIPTION

```diff
@@ -25,7 +25,7 @@ Imports:
     methods,
     mvbutils,
     R6,
-    rmarkdown,
+    rmarkdown(>= 1.9),
```

Comment:
this is a breaking change to `pkgnet` and should be accompanied by a bump in the `pkgnet` version number to `0.2.0`, IMHO